### DISCUSSION TITLE
Welcome to SteamGiftBot Discussions!

### DISCUSSION THREAD

**Discussion by PalmaLuv** [PalmaLuv/SteamGiftBot#6] [Announcements]
## 👋 Welcome to Discussions.
  Hello, this discussion was made for the purpose of improving the software `SteamGiftBot`.  
  Such suggestions are welcome here:
  - Bugs.
  - Suggestions for improvement.

  If there are suggestions to improve the application, then I'm waiting for your suggestions below, regards Palma. :shipit:

**Comment by skrobbi** [comment]
Is it possible to have the bot enter into giveaways for games with cards only?

**Reply by PalmaLuv** [reply]
Yes, I can. I'll try to realize it in my spare time.

**Comment by SonGorLick** [comment]
Question, have you ever considered expanding this to other giveaway sites like indiegala

**Reply by rolegiv** [reply]
It will be really good project if this option will be add in script
